### PR TITLE
Update UI design part2

### LINKED ISSUE #64: Add marign between entries

### PR DESCRIPTION
**Pull request by YangQF2002**
Fixes #64, with extra refinements

### CODE REVIEW

File: src/main/resources/view/ResultDisplay.css [hunk 1]
```diff
@@ -22,3 +22,9 @@
     -fx-background-color: transparent;
 }
 
+#resultDisplay .scroll-bar {
+    -fx-opacity: 0;
+    -fx-pref-width: 1;
+    -fx-max-width: 0;
+    -fx-min-width: 0;
+}
```
Comment (yxtan02) [inline src/main/resources/view/ResultDisplay.css:30]: Perhaps we can modify the scrollbar scrolling speed to be slightly slower to allow for a more pleasant user experience.

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -45,15 +45,15 @@
...
 
-        <SplitPane VBox.vgrow="ALWAYS" styleClass="pane-with-border">
+        <HBox fx:id="contactLists" VBox.vgrow="ALWAYS" styleClass="pane-with-border" spacing="-3">
           <VBox fx:id="guestList" styleClass="pane-with-border" minWidth="200" prefWidth="200" VBox.vgrow="ALWAYS">
             <StackPane fx:id="guestListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
           </VBox>
 
           <VBox fx:id="vendorList" styleClass="pane-with-border" minWidth="200" prefWidth="200" VBox.vgrow="ALWAYS">
             <StackPane fx:id="vendorListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
           </VBox>
```
Comment (slye20) [inline src/main/resources/view/MainWindow.fxml:55]: Nice job in removing SplitPane!

**Review by yxtan02 — COMMENTED** [review]
LGTM. Just a small nit to fix.

File: src/main/resources/view/PersonListCard.fxml [hunk 3]
```diff
@@ -7,30 +7,34 @@
 <?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
 
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="cardPane" fx:id="cardPane" stylesheets="@PersonListCard.css" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (weibinlim) [inline src/main/resources/view/PersonListCard.fxml:13]: Nice job of using the stylesheets field!

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 4]
```diff
@@ -125,6 +136,9 @@ void fillInnerParts() {
         vendorListPanel = new PersonListPanel(logic.getFilteredVendorList(), "Vendors");
         vendorListPanelPlaceholder.getChildren().add(vendorListPanel.getRoot());
 
+        HBox.setHgrow(guestList, Priority.ALWAYS);
+        HBox.setHgrow(vendorList, Priority.ALWAYS);
```
Comment (weibinlim) [inline src/main/java/seedu/address/ui/MainWindow.java:140]: Good job splitting the display of guest and vendor list!

File: src/main/resources/view/PersonListCard.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+.cell_big_label {
+    -fx-font-family: "Arial";
+    -fx-font-size: 16px;
+}
+
+.cell_small_label {
+    -fx-font-family: "Arial";
+    -fx-font-size: 13px;
+}
```
Comment (weibinlim) [inline src/main/resources/view/PersonListCard.css:23]: Should we increase the difference in font size to make it more noticeable?

Comment (slye20) [reply src/main/resources/view/PersonListCard.css:23]: I agree, and perhaps a different font family could be used to increase the user's experience?

**Comment by weibinlim** [comment]
LGTM

**Pull request merged**